rpm/0046-Prioritize-GMP-plugins-over-all-others-and-support-d.patch

```diff
@@ -1,53 +1,53 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Andrew Branson <andrew.branson@jolla.com>
 Date: Wed, 24 Jun 2020 07:56:50 +0200
-Subject: [PATCH] [sailfishos][gecko] Prioritize GMP plugins over all others,
- and support decoding video for h264, vp8 & vp9.
+Subject: [PATCH] Prioritize GMP plugins over all others, and support decoding
```

Comment:
```suggestion
Subject: [PATCH] [sailfishos][gecko] Prioritize GMP plugins over all others, and support decoding
```
Could you keep "[sailfishos][gecko]" so that it's visible from the commit that a commit tampers gecko tree. Apply would keep "[sailfishos][gecko]" when applied like "git am --keep-non-patch"

Reply:
Soon as a I figure out how I removed it in the first place.

Reply:
I'd guess that you applied older patches like (1) "git am --3way rpm/file.patch" on top already fixed patches. Then formatted patch like (2) "git format-patch --zero-commit -N". 

When doing like this, all words at the beginning of each line that are surrounded by square brackets will disappear in step (1).